skt/kernelbuilder.py

```diff
@@ -150,12 +150,12 @@ def getrelease(self):
             self.__prepare_kernel_config()
 
         args = self.make_argv_base + ["kernelrelease"]
-        mk = subprocess.Popen(args, stdout=subprocess.PIPE)
-        (stdout, _) = mk.communicate()
+        proc = subprocess.Popen(args, stdout=subprocess.PIPE)
```

Comment:
I'd prefer the name to be more descriptive/specific - while the `mk` name is definitely not either of them, the first thing some of us will think of when reading `proc` would be the `/proc` directory and not make process.

Reply:
@veruu I am not sure about what `mk` means?  It implies `make kernel` ?

Reply:
It probably means `make` (kernel people have a tendency to ignore vowels when naming things), the name was there before I got involved with the project so I'm not 100% sure.

Reply:
I rename it to `make`